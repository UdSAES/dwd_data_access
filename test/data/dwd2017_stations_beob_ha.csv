ID,Stations-Name,WMO-Kennung,BG,BM,BS,LG,LM,LS,GEOGR_BREITE,GEOGR_LAENGE,STATIONSHOEHE,Betreiber,Melde-Grp,Country
954,UFS Deutsche Bucht,10007,54,10,0,7,27,0,54.1667,7.45,0,DWD,AUT,Germany
2115,Helgoland,10015,54,10,30,7,53,31,54.174957,7.891954,4,DWD,AUT,Germany
1346,Feldberg/Schwarzwald,10908,47,52,30,8,0,14,47.874893,8.003817,1489.6,DWD,AUT,Germany
5792,Zugspitze,10961,47,25,15,10,59,5,47.420868,10.984724,2964,DWD,I Berg,Germany
2290,Hohenpei√üenberg,10962,47,48,3,11,0,39,47.800864,11.010754,977,DWD,I,Germany
1550,Garmisch-Partenkirchen,10963,47,28,59,11,3,44,47.48305,11.062293,719,DWD,AUT,Germany
856,Chieming,10982,47,53,4,12,32,25,47.884339,12.540379,551.2,DWD,AUT,Germany
3484,Neuburg/Donau (Flugplatz),10853,48,42,35,11,12,35,48.709746,11.209617,380,GeoInfoDBw,I G,Germany
50036456,GEORG VON NEUMAYER,89002,70,40,23,8,16,30,-70.673054,-8.27498,40,AWI,oZ,Antarctica
