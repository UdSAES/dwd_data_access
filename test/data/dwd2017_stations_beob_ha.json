[ {
    "id": 954,
    "stations-name": "UFS Deutsche Bucht",
    "wmo-kennung": 10007,
    "location": {
      "latitude": 54.1667,
      "longitude": 7.45,
      "elevation": 0
    },
    "betreiber": "DWD",
    "melde-grp": "AUT",
    "country": "Germany"
  },
  {
    "id": 2115,
    "stations-name": "Helgoland",
    "wmo-kennung": 10015,
    "location": {
      "latitude": 54.174957,
      "longitude": 7.891954,
      "elevation": 4
    },
    "betreiber": "DWD",
    "melde-grp": "AUT",
    "country": "Germany"
  },
  {
    "id": 1346,
    "stations-name": "Feldberg/Schwarzwald",
    "wmo-kennung": 10908,
    "location": {
      "latitude": 47.874893,
      "longitude": 8.003817,
      "elevation": 1489.6
    },
    "betreiber": "DWD",
    "melde-grp": "AUT",
    "country": "Germany"
  },
  {
    "id": 5792,
    "stations-name": "Zugspitze",
    "wmo-kennung": 10961,
    "location": {
      "latitude": 47.420868,
      "longitude": 10.984724,
      "elevation": 2964
    },
    "betreiber": "DWD",
    "melde-grp": "I Berg",
    "country": "Germany"
  },
  {
    "id": 2290,
    "stations-name": "Hohenpei√üenberg",
    "wmo-kennung": 10962,
    "location": {
      "latitude": 47.800864,
      "longitude": 11.010754,
      "elevation": 977
    },
    "betreiber": "DWD",
    "melde-grp": "I",
    "country": "Germany"
  },
  {
    "id": 1550,
    "stations-name": "Garmisch-Partenkirchen",
    "wmo-kennung": 10963,
    "location": {
      "latitude": 47.48305,
      "longitude": 11.062293,
      "elevation": 719
    },
    "betreiber": "DWD",
    "melde-grp": "AUT",
    "country": "Germany"
  },
  {
    "id": 856,
    "stations-name": "Chieming",
    "wmo-kennung": 10982,
    "location": {
      "latitude": 47.884339,
      "longitude": 12.540379,
      "elevation": 551.2
    },
    "betreiber": "DWD",
    "melde-grp": "AUT",
    "country": "Germany"
  },
  {
    "id": 3484,
    "stations-name": "Neuburg/Donau (Flugplatz)",
    "wmo-kennung": 10853,
    "location": {
      "latitude": 48.709746,
      "longitude": 11.209617,
      "elevation": 380
    },
    "betreiber": "GeoInfoDBw",
    "melde-grp": "I G",
    "country": "Germany"
  },
  {
    "id": 50036456,
    "stations-name": "GEORG VON NEUMAYER",
    "wmo-kennung": 89002,
    "location": {
      "latitude": -70.673054,
      "longitude": -8.27498,
      "elevation": 40
    },
    "betreiber": "AWI",
    "melde-grp": "oZ",
    "country": "Antarctica"
  }
]
