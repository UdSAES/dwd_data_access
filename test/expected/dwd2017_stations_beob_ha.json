[
  {
    "location": {
      "latitude": 54.1667,
      "longitude": 7.45,
      "elevation": 0
    },
    "ID": "954",
    "Stations-Name": "UFS Deutsche Bucht",
    "WMO-Kennung": "10007",
    "Betreiber": "DWD",
    "Melde-Grp": "AUT",
    "Country": "Germany"
  },
  {
    "location": {
      "latitude": 54.174957,
      "longitude": 7.891954,
      "elevation": 4
    },
    "ID": "2115",
    "Stations-Name": "Helgoland",
    "WMO-Kennung": "10015",
    "Betreiber": "DWD",
    "Melde-Grp": "AUT",
    "Country": "Germany"
  },
  {
    "location": {
      "latitude": 47.874893,
      "longitude": 8.003817,
      "elevation": 1489.6
    },
    "ID": "1346",
    "Stations-Name": "Feldberg/Schwarzwald",
    "WMO-Kennung": "10908",
    "Betreiber": "DWD",
    "Melde-Grp": "AUT",
    "Country": "Germany"
  },
  {
    "location": {
      "latitude": 47.420868,
      "longitude": 10.984724,
      "elevation": 2964
    },
    "ID": "5792",
    "Stations-Name": "Zugspitze",
    "WMO-Kennung": "10961",
    "Betreiber": "DWD",
    "Melde-Grp": "I Berg",
    "Country": "Germany"
  },
  {
    "location": {
      "latitude": 47.800864,
      "longitude": 11.010754,
      "elevation": 977
    },
    "ID": "2290",
    "Stations-Name": "Hohenpei√üenberg",
    "WMO-Kennung": "10962",
    "Betreiber": "DWD",
    "Melde-Grp": "I",
    "Country": "Germany"
  },
  {
    "location": {
      "latitude": 47.48305,
      "longitude": 11.062293,
      "elevation": 719
    },
    "ID": "1550",
    "Stations-Name": "Garmisch-Partenkirchen",
    "WMO-Kennung": "10963",
    "Betreiber": "DWD",
    "Melde-Grp": "AUT",
    "Country": "Germany"
  },
  {
    "location": {
      "latitude": 47.884339,
      "longitude": 12.540379,
      "elevation": 551.2
    },
    "ID": "856",
    "Stations-Name": "Chieming",
    "WMO-Kennung": "10982",
    "Betreiber": "DWD",
    "Melde-Grp": "AUT",
    "Country": "Germany"
  },
  {
    "location": {
      "latitude": 48.709746,
      "longitude": 11.209617,
      "elevation": 380
    },
    "ID": "3484",
    "Stations-Name": "Neuburg/Donau (Flugplatz)",
    "WMO-Kennung": "10853",
    "Betreiber": "GeoInfoDBw",
    "Melde-Grp": "I G",
    "Country": "Germany"
  },
  {
    "location": {
      "latitude": -70.673054,
      "longitude": -8.27498,
      "elevation": 40
    },
    "ID": "50036456",
    "Stations-Name": "GEORG VON NEUMAYER",
    "WMO-Kennung": "89002",
    "Betreiber": "AWI",
    "Melde-Grp": "oZ",
    "Country": "Antarctica"
  }
]
